### ISSUE TITLE
Add container support for EC2 ImageBuilder

### ISSUE THREAD

**Issue by breathingdust** [hashicorp/terraform-provider-aws#23234]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### New or Affected Resource(s)

* aws_imagebuilder_container_recipe

### Potential Terraform Configuration

```hcl
resource "aws_imagebuilder_container_recipe" "example" {
  container_type = "DOCKER"

  component {
    component_arn = ""
    parameter {
      name  = "Parameter1"
      value = "Value1"
    }
  }
  name = "example"
  description = "example"
  dockerfile_template_data = ""
  dockerfile_template_uri = ""
  target_repository = ""
  working_directory = ""
  semantic_version = "1.0.0"
  parent_image = ""
}
```

### References

* [CreateContainerRecipe API](https://docs.aws.amazon.com/imagebuilder/latest/APIReference/API_CreateContainerRecipe.html#API_CreateContainerRecipe_RequestSyntax)

**Comment by kamilturek** [comment]
Hey @breathingdust, I believe this resource is already implemented. Please see https://github.com/hashicorp/terraform-provider-aws/issues/16839.

**Comment by breathingdust** [comment]
Thanks @kamilturek! Will close.

This was shipped in `v4.1.0` of the provider
